benchmarks/400.inference/413.image-classification/python/function.py

```
@@ -0,0 +1,178 @@
+    """
+    Lazily download, extract, and initialize the ONNX ResNet model.
+    """
+    global _session, _session_input, _session_output, _cached_model_key
+
+    effective_model_key = model_key or MODEL_ARCHIVE
+    model_download_begin = datetime.datetime.now()
+    model_download_end = model_download_begin
+
+    if _session is None or _cached_model_key != effective_model_key:
+        archive_basename = os.path.basename(effective_model_key)
+        archive_path = os.path.join("/tmp", f"{uuid.uuid4()}-{archive_basename}")
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)